docs/reference/server-configuration.md

```diff
@@ -118,6 +118,13 @@ Controls the semantics of the SUBSTRING UDF. Refer to the SUBSTRING
 documentation in the [function](/developer-guide/ksqldb-reference/scalar-functions)
 guide for details.
 
+## `ksql.heartbeat.enable`
+
+If enabled, ksqlDB servers in the same ksqlDB cluster send heartbeats to each
+other, to aid in faster failure detection for improved pull query routing.
```

Comment:
I'm not sure how we want to tie this into the greater HA functionality messaging.  This is useful largely if we can fallback on a standby (in which case it must be used with `ksql.query.pull.enable.standby.reads` and `ksql.streams.num.standby.replicas`), though technically, it can be used to fail more quickly if used by itself.  It might be useful to mention those other flags in this context.

Reply:
Thanks @AlanConfluent , this is really helpful context. I think it makes sense to have a separate docs topic on HA functionality, rather than trying to get into details in the config reference. How about I file a docs ticket to add a new topic on HA pull queries, and including updating the blurbs for these configs (and adding the other two you mentioned) as part of that? I'd like to merge this PR first without the more in-depth coverage on HA, as that'd be a rather large expansion of scope for this PR.